lib/vault/client.rb

```diff
@@ -16,6 +16,9 @@ class Client
     # The name of the header used to hold the Vault token.
     TOKEN_HEADER = "X-Vault-Token".freeze
 
+    # The name of the header used to hold the wrapped request ttl.
+    WRAP_TTL_HEADER = "X-Vault-Wrap-Ttl".freeze
```

Comment:
Good catch.  Looks like the http proxy I was using for equivalence testing with the go client did some case manipulation.